### DISCUSSION TITLE
A way to disable "Screen Edges" shortcuts ?

### DISCUSSION THREAD

**Discussion by SpidFightFR** [nix-community/plasma-manager#426] [Q&A] [answered]
Hey there !

Is there a way to disable all the actions triggered by plasma when the cursor reaches a corner of the screen or its edges, please ?

**Comment by HeitorAugustoLN** [comment, marked as answer]
Well, if it is the default one that plasma has, you could remove it with:
```nix
{
  programs.plasma.configFile.kwinrc.Effect-overview.BorderActivate = 9;
}
```

If it is not you should try modifying it manually and see the reflected changes in `~/.config/kwinrc`. And write them in the nix configuration

I was going to add some configuration for screen edges in #308, but I never got the free time to do so. Maybe one day, I finish it.

**Reply by SpidFightFR** [reply]
thanks man !

**Reply by HeitorAugustoLN** [reply]
No problem :)
